x-pack/legacy/plugins/license_management/public/np_ready/application/lib/es.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { HttpSetup } from 'src/core/public';
+
+export function putLicense(http: HttpSetup, license: string, acknowledge: boolean) {
+  return http.put('/api/license', {
```

Comment:
nit: it might be helpful to create a constant for the api base path

Reply:
👍